rholang/src/main/scala/coop/rchain/rholang/interpreter/registry/RegistryBootstrap.scala

```diff
@@ -1,4 +1,4 @@
-package coop.rchain.casper.genesis.contracts
+package coop.rchain.rholang.interpreter.registry
```

Comment:
can you fix the commit log to include the moved version in the first commit?

Reply:
Done